site/content/getting-started/javascript-api.md

```diff
@@ -22,17 +22,35 @@ For example, the [Ticker]({{< docsref "/components/ticker" >}}) component would
 <div class="ticker" data-of-ticker></div>
 ```
 
-### Manual initialization
+However, the component elements may be added to the DOM after the `DOMContentLoaded` event is fired (e.g., via Ajax request). In that case, OpenFrontend will *not* initialize them automatically.
+
+There are two ways to tackle this problem: initialize components semi-automatically or manually.
+
+### Semi-automatic initialization
+
+You can call the global method `initOpenFrontend(element)`, which will traverse through all elements inside the container and initialize the components just like it would do on the `DOMContentLoaded` event.
+
+For example, if you would like to initialize components on a newly added HTML buffer:
+
+```js
+const container = document.getElementById('#ajax-container')
+
+// Add new elements to the DOM
+container.innerHTML = await fetch('endpoint/buffer').then(r => r.text())
 
-It may happen that the component elements are added to the DOM after the `DOMContentLoaded` event is fired. In that case, OpenFrontend will *not* initialize them automatically.
+// Initialize OpenFrontend for newly added elements
+window.initOpenFrontend(container)
+```
+
+### Manual initialization
 
-In that case, you have to access the API yourself and initialize the components manually.
+You can also initialize the components yourself. This might be useful if you need to set custom options or interact with the component directly.
```

Comment:
```suggestion
You can also initialize the components yourself. This might be useful if you need to set dynamic (e.g. based on other JavaScript logic) options or interact with the component directly.
```

Only setting custom options would be easier using the `data` attribute 😉